src/posts/en/configuring-babel-6-for-node-js.md

```diff
@@ -0,0 +1,164 @@
+
+In order to do that we'll install two presets as part of our devDependencies
+- [es2015](https://babeljs.io/docs/plugins/preset-es2015/)
+- [stage-0](https://babeljs.io/docs/plugins/preset-stage-0/)
+
+Type the following command in order to install these presets.
+```bash
+npm install --save-dev babel-preset-es2015 babel-preset-stage-0
+```
+In order to access a wide range of babel plug ins click [here](https://babeljs.io/docs/plugins/)
+
+You need to include these presets in command you issue
+```bash
+babel --presets es2015,stage-0 code/index.js -o build/app.js
+```
+
+You will see normal es5 code generated in `app.js`, this is called **Transpiled code** (a term used widely in JS World). You can run this code using the command below.
+```bash
+node build/app.js
+```
+
+### Setting up a proper build environment using babel
+This is all good magic but what about doing some proper development using node.js.
+
+#### babel configuration file .babelrc
+`.babelrc` is a very neat way to separate all your babel stuff in one JSON file. It's also pretty easy to get started. This is our .babelrc file for this tutorial. You can access other .babelrc options [here](http://babeljs.io/docs/usage/options/) and make it as complex as you want it to be
+```json
+{
+  "plugins": ["es2015", "stage-0"]
+}
+```
+This is pretty much it for this tutorial, this will help us with our rest of development. Now whenever we want to add or subtract plugins. Instead of changing the command we will change the plugins array in this file. Easy! Isn't it?
+
+Now if you run
+```bash
+babel -w code/ -d build/
+```
+It will read the **presets** to use from `.babelrc` compile the code in `code/` directory and generate the compiled code javascript files in `build/` folder. But hey! The command didn't end. Notice the `-w` flag, **w** stands for **watch**, it will recompile the code as you make changes in your code directory. COOL! Now this is some magic I am talking about.
+
+#### Using Source maps in your file
+If you are thinking that's all cool and fun but what about syntax highlighting and real time code debugging. You don't have to be worried. Source maps are just for this purpose. Source maps tell node.js that this code is transpiled and let's you highlight **source** file instead of the **transpiled** file 😄. YAYY!
+
+This file here `code/error.js` throws an error after the second yield in the generator but the transpiled code doesn't quite look like this.
+```js
+function* errorFulGenerator() {
+  yield "yo"
+  throw new Error("source maps are awesome")
+  return ""
+}
+
+var errorGen = errorFulGenerator()
+errorGen.next()
+errorGen.next()
+```
+
+We use this command to generate **source maps** along with the **transpiled** code *notice the `--source-maps flag`*
+```bash
+babel code/ -d build/ --source-maps
+```
+
+Now when we encounter the error we get useful debugging such as this
+```js
+errorGen.next()
+         ^
+Error: source maps are awesome
```

Comment:
I think it would be nice if you added the error's call stack as well—that's basically the main point of source maps, making the call stack point to the lines in the original sources instead of the transpiled ones.